content/concepts/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Introduction
+---
+
+*webpack* is a _module_ _bundler_ for modern JavaScript applications.
+
+### What is a module?
+
+In [modular programming](https://en.wikipedia.org/wiki/Modular_programming), developers break programs up into discrete chunks of functionality called a _module_.
+
+Each module has a smaller surface area than a full program, making verification, debugging, and testing trival. 
+Well-written _modules_ provide solid abstractions and encapsulation boundaries, so that each module has a coherent design and a clear purpose within the overall application.
+
+Node.js has supported modular programming almost since its inception. 
+On the web, however, support for _modules_ has been slow to arrive.
+Multiple tools exist that support modular JavaScript on the web, with a variety of benefits and limitations.
+webpack builds on lessons learned from these systems and applies the concept of _modules_ to any file in your project. 
+
+### What is a webpack module?
+
+In contrast to Node.js modules, webpack _modules_ can express their _dependencies_ in a variety of ways. A few examples are:
+
+* A JavaScript `require()` statement
+* An ECMAScript2015 `import` statement [<sup>[1]</sup>](#1)
+* An AMD `define` and `require` statement
+* An `@import` statement inside of a css/sass/less file.
+* An image url in a stylesheet or html file. 
+
+### webpack, dependency graph, and bundles
+
+Any time one file depends on another (as seen above), webpack treats this as a _dependency_. This allows webpack to take non-code assets, such as images or web fonts, and also provide them as _dependencies_ for your application.
```

Comment:
Looking at @bebraw comment "Maybe it's good to expand on the idea of the module, not just code." on an earlier commit, I feel like you over-corrected this statement. I feel that it should reference code, but also reference non-code assets as well.